main.js

```diff
@@ -185,23 +185,34 @@ function openCommentTextFn() {
 // }
 
 
-function createPost() {
+
+
+// Creating border Phone
+const borderPhone = document.createElement('div');
+borderPhone.className = "phone";
```

Comment:
these two lines are repeating quite often, therefore, create a fn that will create an element given its tag name (pass this as a parameter), it should return newly created element